platform-ajax/src/main/java/com/softicar/platform/ajax/document/creation/AjaxDocumentPagePrinter.java

```diff
@@ -86,6 +86,7 @@ private void printJavascriptFramework() {
 		// texts
 		println("var LOCK_MESSAGE = '%s';", AjaxI18n.PLEASE_WAIT.concat(" ").concat(AjaxI18n.ANOTHER_ACTION_IS_STILL_EXECUTING));
 		println("var AUTO_COMPLETE_TEXT_GENERIC = '%s';", AjaxI18n.AUTO_COMPLETE_INPUT_ELEMENT);
+		println("var AUTO_COMPLETE_TEXT_FURTHER_ENTRIES_AVAILABLE = '%s';", AjaxI18n.FURTHER_ENTRIES_AVAILABLE);
```

Comment:
Now that I think about this:
We could have just defined another new variable here which represents `DomAutoCompleteList.MAXIMUM_ELEMENT_COUNT` and use it in the javascript.
With that, there would be way less code to adapt than the current approach.